src/server/string_family.cc

```diff
@@ -1170,6 +1259,63 @@ auto StringFamily::OpMGet(bool fetch_mcflag, bool fetch_mcver, const Transaction
   return response;
 }
 
+void StringFamily::ClThrottle(CmdArgList args, ConnectionContext* cntx) {
+  string_view key = ArgS(args, 1);
+
+  int64_t max_burst;
+  string_view max_burst_str = ArgS(args, 2);
+  if (!absl::SimpleAtoi(max_burst_str, &max_burst)) {
```

Comment:
does it make that max_burst is signed? I would expect that it would be unsigned.